### ISSUE TITLE
Tests fail with Dancer2 0.200000

### ISSUE THREAD

**Issue by eserte** [Grinnz/Dancer2-Serializer-JSONMaybeXS#1]
My smokers started to report the following failure:

```
#   Failed test 'Correct error message'
#   at t/deserialize.t line 171.
#                   'Failed to deserialize content: malformed number (no digits after initial minus), at character offset 1 (before "--") at /home/cpansand/.cpan/build/2016060906/Dancer2-Serializer-JSONMaybeXS-0.001-JlU5wp/blib/lib/Dancer2/Serializer/JSONMaybeXS.pm line 30.
# '
#     doesn't match '(?^x:
#                 ^
#                 Failed\ to\ deserialize\ the\ request\:\ 
#                 malformed\ number
#             )'
# Looks like you failed 1 test of 13.
t/deserialize.t .......... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/13 subtests 
```

Statistical analysis suggests that the problem is caused by the newest Dancer2 version:

```
****************************************************************
Regression 'mod:Dancer2'
****************************************************************
Name                   Theta          StdErr     T-stat
[0='const']           1.0000          0.0000    25845901939911908.00
[1='eq_0.165000']             0.0000          0.0000       1.42
[2='eq_0.166000']             0.0000          0.0000       2.03
[3='eq_0.166001']             0.0000          0.0000       0.00
[4='eq_0.200000']            -1.0000          0.0000    -24905734296884396.00

R^2= 1.000, N= 70, K= 5
****************************************************************
```

**Comment by amurf** [comment]
Looks like an error message has been re-worded.

See here: https://github.com/PerlDancer/Dancer2/commit/f8a05e41dc42ab984067786e844ffcd624f65f1c

**Comment by Grinnz** [comment]
Thanks for the report. Released as 0.002.
